docker-compose.yaml

```diff
@@ -13,7 +13,7 @@ services:
       - "5432:5432"
 
   redis:
-    image: redis:6.0
+    image: redis:6.2
```

Comment:
Does this algorithm require features only available >= 6.2?

Reply:
Yes, `PXAT` is added at 6.2